runtime/hydradx/src/benchmarking/omnipool.rs

```diff
@@ -433,6 +436,128 @@ runtime_benchmarks! {
 		assert!(asset_state.cap == 100_000_000_000_000_000u128);
 	}
 
+	trade_execution_sell {
```

Comment:
What is the reason that these two tests are not placed in the omnipool pallet?

Reply:
These are benchmarks. The weights are used in the `AmmWeights` struct.

Reply:
Yes, but the similar `trade_execution_sell` and `trade_execution_buy` of LBP are placed to the LBP pallet `benchmark.rs` file. 

So why not putting these then into the omnipool pallet?

Reply:
We don't have benchmarks.rs in the omnipool pallet.

Reply:
dependencies reasons probably